src/components/Pagination/PaginationButton/PaginationButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from "react";
+import { useLanguage } from "@/context/LocaleProvider";
+import { PaginationButtonProps } from "./types";
+import { StyledButton, StyledIconWrapper, StyledText, StyledWrapper } from "./styles";
+
+const PaginationButton: FC<PaginationButtonProps> = ({ intent, onClick, icon, isVisible }) => {
```

Comment:
I'm not sure about structure of this component - we have 30 lines here and the only difference between variants is the label. Am I missing something?